.github/workflows/repo2docker.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     - name: Build pudl-jupyter docker image with repo2docker
```

Comment:
Are you keeping the repo2docker action around if folks want to deploy the package in jupyter lab?

Reply:
I'm thinking of this repository as kind of a reference for how we do different CI/infra things. So right now we're building an image using `repo2docker` for the `pudl-examples` repo that we use on the JupyterHub, and we're also building an image directly, and it seemed like having both of those in brief examples would be helpful.